examples/misc/upload_media_bundle.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example uploads an HTML5 zip file as a media bundle."""
+
+
+import argparse
+import sys
+import requests
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+BUNDLE_URL = 'https://goo.gl/9Y7qI2'
+
+def main(client, customer_id):
+    """Main method, to run this code example as a standalone application."""
```

Comment:
Don't necessarily need this pydoc since it doesn't really add any value, don't need the empty line following it.

Reply:
fixed